omod/src/main/webapp/resources/scripts/components/EncounterFactory.js

```diff
@@ -194,11 +194,13 @@ angular.module("encounterFactory", [])
          @return The WebService object
          */
         Encounter.toObsWebServiceObject = function (conceptUuid, obsValue, existingObsUuid) {
-            var ret = {concept: conceptUuid, value: null, uuid: null};
+            var ret = {};
+            if (conceptUuid != null) {
+              ret.concept = conceptUuid;
+            }
```

Comment:
What's the point of all this? Why not just

```js
return { concept: conceptUuid, value: obsValue, uuid: existingObsUuid }
```
?

Unless something downstream is differentiating between `undefined` and `null`?

Reply:
This is being used to generate a REST request to send to OpenMRS, so I did want to prefer not including a null property into the response at all.